packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -36,7 +36,7 @@ async function buildSwingset(
   { consensusMode, debugName = undefined, slogCallbacks, slogFile },
 ) {
   const debugPrefix = debugName === undefined ? '' : `${debugName}:`;
-  let config = loadSwingsetConfigFile(vatconfig);
+  let config = await loadSwingsetConfigFile(vatconfig);
```

Comment:
This is a consequence of the breaking change to the device plugin API, which now needs to use async module specifier resolution.